ingresspipe/models/metadata.py

```diff
@@ -0,0 +1,262 @@
+import pandas as pd
+import networkx as nx
+import json
+import re
+
+from jsonschema import Draft7Validator, exceptions, validate, ValidationError
+
+# allows specifying explicit variable types
+from typing import Any, Dict, Optional, Text, List
+
+# handle schema logic; to be refactored as SchemaExplorer matures into a package
+# as collaboration with Biothings progresses
+
+import sys
+sys.path.append("..") # Adds higher directory to python modules path.
```

Comment:
This is a pretty common pattern for running and testing a Python package _before_ you've created your `setup.py` to make it installable. With that working, you can use `pip[3] install -e .` to install the package in 'developer' mode, which allows you to keep making changes to the code while being able to run like any other installed Python library.

Unless you're accessing files outside the main package folder (e.g., configs, data, etc.), you shouldn't need this line.

Reply:
Yup, you're right. I added the setup.py file (i.e., made the app installable) and tested it without the above lines, and it works without the need to add `..` into the modules PATH.

Thank you!